yoti_python_sdk/issuance_details.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+from yoti_python_sdk import date_parser
+import base64
+
+
+class IssuanceDetails(object):
+    def __init__(self, data_entry):
+        self.__token = base64.b64encode(data_entry.issuance_token)
+        self.__expiry_date = date_parser.datetime_with_microsecond(
+            data_entry.issuing_attributes.expiry_date
+        )
+        self.__attributes = data_entry.issuing_attributes.definitions
```

Comment:
`__attributes` is `<class 'google.protobuf.pyext._message.RepeatedCompositeContainer'>`

Should this instead be a list of `AttributeDefinition` (or list of dictionary `{"name": "someName"}`)?

Reply:
I can see this is now `<class 'list'>` of `IssuingAttributes_pb2.Definition` - should this be a list of dictionary?

Reply:
Oh. You're right. I just checked the type of the first :(